vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -416,30 +465,29 @@ func (d *VolumeDriver) Mount(r volume.MountRequest) volume.Response {
 // Unmount request from Docker. If mount refcount is drop to 0.
 // Unmount and detach from VM
 func (d *VolumeDriver) Unmount(r volume.UnmountRequest) volume.Response {
-	log.WithFields(log.Fields{"name": r.Name}).Info("Unmounting Volume ")
+	log.WithField("name", r.Name).Info("Unmounting Volume")
 
 	// if the volume is still used by other containers, just return OK
 	refcnt, err := d.decrRefCount(r.Name)
 	if err != nil {
 		// something went wrong - yell, but still try to unmount
-		log.WithFields(
-			log.Fields{"name": r.Name, "refcount": refcnt},
-		).Error("Refcount error - still trying to unmount...")
+		log.WithFields(log.Fields{
+			"name":     r.Name,
+			"refcount": refcnt,
+		}).Error("Refcount error - still trying to unmount...")
 	}
-	log.Debugf("volume name=%s refcnt=%d", r.Name, refcnt)
+
 	if refcnt >= 1 {
-		log.WithFields(
-			log.Fields{"name": r.Name, "refcount": refcnt},
-		).Info("Still in use, skipping unmount request. ")
+		log.WithFields(log.Fields{
+			"name":     r.Name,
+			"refcount": refcnt,
+		}).Debug("volume still in used; skipping unmount request")
```

Comment:
Hi @govint,

Oops, mea culpa. 

Please note I am going to stop responding to your comments. You should stop reviewing this PR. I closed it. So please, stop :)